Makefile.validation

```diff
@@ -37,7 +37,7 @@ pr-validation-junit:
 # Run e2e tests for CI. All of these tests use ginkgo so we can use native ginkgo methods to print junit output.
 # Also go test doesn't provide functionnality to save test's log into the file. Use > to archieve artifatcs.
 test-ci:
-	${GO_ENV_VARS} CI=true CSI_VERSION=${CSI_VERSION} OPERATOR_VERSION=${OPERATOR_VERSION} go test -v test/e2e/baremetal_e2e_test.go -ginkgo.v -ginkgo.progress -kubeconfig=${HOME}/.kube/config -chartsDir ${CHARTS_DIR} -timeout=0 > log.txt
+	${GO_ENV_VARS} CI=true CSI_VERSION=${CSI_VERSION} OPERATOR_VERSION=${OPERATOR_VERSION} go test -v test/e2e/baremetal_e2e_test.go -ginkgo.v -ginkgo.progress -ginkgo.skip='Baremetal-csi' -kubeconfig=${HOME}/.kube/config -chartsDir ${CHARTS_DIR} -timeout=0 
```

Comment:
You should update your fork and use `test-short-ci`, not editing the existing one